.yardopts

```diff
@@ -6,4 +6,5 @@
 --hide-api private
 --plugin sitemap
 --exclude \/client_api.rb$
-gems/**/lib/**/*.rb
+gems/aws-sdk-core/lib/**/*.rb
```

Comment:
Assume this is just for testing and should be reverted before merge?

Reply:
Yeah. It would be reverted. You can use it to build s3 docs and see the changes for yourself (please do so)